### ISSUE TITLE
Megasloth281093 ingested destroyed thing Plant_Grass316472

### ISSUE THREAD

**Issue by UndeadKitten** [cseelhoff/RimThreaded#405]
**Describe the bug**
Another red error that caused the log to popup, no noticeable issues in-game. I was playing the same (new) save game as my last 5 posts, with a freshly installed Rimworld and config folder. I only have Harmony and RimThreaded installed, and I don't have Royalty DLC. The total worker threads is set to 4. I'll mention that I restart my game after each new unique error to submit a report.
EDIT: Version was RT 1.2.3

**Error Log**
https://gist.github.com/UndeadKitten/67470b5505e807ee7f50b3afad2a9ca7

**Specs**
Operating System: Windows 10 Home 64-bit
Processor: Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz (4 CPUs), ~3.5GHz
Memory: 16384MB RAM
Card name: NVIDIA GeForce GTX 970

**Comment by pastorismylord** [comment]
any advice on how to reproduce this error? do you have a save available for this error?

**Comment by bababooeyy** [comment]
i think this is just multiple threads calculating the same job, thus the 'tried to reserve job Ingest (Job_560306) A=Thing_Plant_Grass316472 on target Thing_Plant_Grass316472, but it's already reserved by him.', and the 'ingested destroyed thing Plant_Grass316472'.
Also don't think it's reliably reproducible, even with a save, because it *might* be caused by the threading thing.
Tried spawning in something like 300 muffaloes and lots of grass, can get the error pretty fast on 3x speed just waiting.
Just gotta pray to Randy that your threads dont clash, i guess

**Comment by UndeadKitten** [comment]
This is one of the many minor errors that I have encountered, there are many different ones and I don't know what causes them so I have not tried reproducing any of them.

(Below is copy and pasted across a few of my posts)

I don't know of a way to reliably reproduce any of the errors that I get, they happen seemingly at random while playing. Sometimes almost immediately upon loading up the save and sometimes hours later. I get many different errors, more than what I have posted here. I regularly get inconsequential errors and sometimes ones that break functions of the game (usually pawn unable to reach area) that are fixed upon restarting the game. Both on 1.2.3 and 1.2.4 preview.

If you have any suggestions for me on what to do then let me know.
